### PR TITLE
amending installed package list

### PR DESCRIPTION
**Pull request by JasonBellONS**
amending installed package list to correctly install cython for pandas installation as a subpackage

It was noted that python was upgraded from 3.12.7 to 3.13 on the underlying container, this has caused issues with pandas and jsonnet. Pandas has an upgrade for its package which has been fixed but jsonnet has no update as of yet, package removed and will be replaced with the json package in a second PR)

### CODE REVIEW

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by gibbardsteve** [comment]
It looks like CICD is bailing still...
image

**Comment by JasonBellONS** [comment]
> It looks like CICD is bailing still... image

seems to be due to python on the containers now suddenly being 3.13 of python (previously 3.12.7). If i cant get it working i may need to speak to shared services about getting a temporary version of the container created with 3.12.7 again

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dc2kio1yw8kuv.cloudfront.net/
cloudfront_id=EKBNK1K1QJGP1

**Comment by JasonBellONS** [comment]
Method and glossary amended to remove jsonnet package (note this breaks non contentful runs, a second branch will be created to implement regular .json files for these two pages)

File: sml_builder/method.py [hunk 1]
```diff
@@ -183,11 +139,11 @@ def display_methods():
             cms_enabled=content_management["enabled"],
         )
 
-    methods_dir = "./content/methods/ready-to-use-methods"
-    future_methods_dir = "./content/methods/future-methods"
+    # methods_dir = "./content/methods/ready-to-use-methods"
```
Comment (jhmjam) [inline sml_builder/method.py:142]: delete these?

File: sml_builder/method.py [hunk 2]
```diff
@@ -200,23 +156,23 @@ def display_methods():
...
-                "language": method["method_metadata"]["Languages"],
-            }
-        )
-    if filter_methods is not None:
-        filtered_methods = [
-            method for method in methods if method["id"] in filter_methods
-        ]
-        methods = filtered_methods
-    return methods
+# def appendRow(methods_dir, filter_methods=None):
```
Comment (jhmjam) [inline sml_builder/method.py:159]: and these?

**Review by jhmjam — CHANGES_REQUESTED** [review]

**Review by jhmjam — APPROVED** [review]

**Pull request merged**